Resources/views/Collector/db.html.twig

```diff
@@ -193,8 +197,14 @@
                                     {{ (query.sql ~ ';')|doctrine_replace_query_parameters(query.params)|doctrine_pretty_query(highlight_only = true) }}
                                 </div>
 
+                                <div id="execute-{{ i }}-{{ loop.parent.loop.index }}" class="sql-runnable hidden">
```

Comment:
if the query is not explainable, it will not be runnable either (a non-explainable query means that we lost some info when serializing parameters, and so we cannot execute it again)